### PR TITLE
Realistic names for APEX content

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Add realistic names for APEX content

Note:
Both LSVs seem to look very similar to existing vehicles (namely the [Polaris Defense DAGOR](http://soldiersystems.net/blog1/wp-content/uploads/2014/10/WDP_8727-Front-Three-Quarters-mid-hight_Wht.jpg) and the [LSV Mk.II](http://www2.tnp.sg/sites/default/files/styles/medium/public/01c9c2cb.jpg?itok=dRpONFAw)), but they're not quite the same so they keep their original name.

*I would like some feedback on the translation of "Green Hex".*

### CODE REVIEW

**Comment by licht-im-Norden87** [comment]
Ich würde das Hex einfach weglassen. Ich denke, dies führt nur zur Verwirrung

**Comment by commy2** [comment]
For "Green" they use the key `grn`
For "Green Hex" they use `ghex`

If we choose the same name for both color shemes, the names will collide once they make a weapon or vehicle with both textures.

**Comment by commy2** [comment]
>Omnirock commented 18 minutes ago
>The civilian Cessna looks to be based on the Corvalis TTx (Cessna 400) model. Basing this on the front intakes and the more angular design as well as wheel/landing gear design.

**Comment by BaerMitUmlaut** [comment]
The manufacturer calls it Cessna TTx which is how I named it in the PR:
http://cessna.txtav.com/en/piston/cessna-ttx

**Comment by Omnirock** [comment]
translations for jeep wrangler, some of the german and russian translations seem to be switched.

File: addons/realisticnames/stringtable.xml [hunk 1]
```diff
@@ -1920,5 +1920,425 @@
...
+            <Portuguese>Jeep Wrangler (Preto)</Portuguese>
+        </Key>
+        <Key ID="STR_ACE_RealisticNames_C_Offroad_02_unarmed_blue">
+            <English>Jeep Wrangler (Blue)</English>
+            <Czech>Jeep Wrangler (Blau)</Czech>
+            <French>Jeep Wrangler (Azul)</French>
+            <Spanish>Jeep Wrangler (Niebieski)</Spanish>
+            <Russian>Jeep Wrangler (Modrý)</Russian>
+            <Polish>Jeep Wrangler (Bleue)</Polish>
+            <German>Jeep Wrangler (Синий)</German>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: This is Russian

File: addons/realisticnames/stringtable.xml [hunk 2]
```diff
@@ -1920,5 +1920,425 @@
...
+            <Russian>Jeep Wrangler (Чёрный)</Russian>
+            <Polish>Jeep Wrangler (Czarny)</Polish>
+            <German>Jeep Wrangler (Schwarz)</German>
+            <Italian>Jeep Wrangler (Nero)</Italian>
+            <Hungarian>Jeep Wrangler (Fekete)</Hungarian>
+            <Portuguese>Jeep Wrangler (Preto)</Portuguese>
+        </Key>
+        <Key ID="STR_ACE_RealisticNames_C_Offroad_02_unarmed_blue">
+            <English>Jeep Wrangler (Blue)</English>
+            <Czech>Jeep Wrangler (Blau)</Czech>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: This is German

File: addons/realisticnames/stringtable.xml [hunk 3]
```diff
@@ -1920,5 +1920,425 @@
...
+            <French>Jeep Wrangler (Azul)</French>
+            <Spanish>Jeep Wrangler (Niebieski)</Spanish>
+            <Russian>Jeep Wrangler (Modrý)</Russian>
+            <Polish>Jeep Wrangler (Bleue)</Polish>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: French ???

File: addons/realisticnames/stringtable.xml [hunk 4]
```diff
@@ -1920,5 +1920,425 @@
...
+            <German>Jeep Wrangler (Schwarz)</German>
+            <Italian>Jeep Wrangler (Nero)</Italian>
+            <Hungarian>Jeep Wrangler (Fekete)</Hungarian>
+            <Portuguese>Jeep Wrangler (Preto)</Portuguese>
+        </Key>
+        <Key ID="STR_ACE_RealisticNames_C_Offroad_02_unarmed_blue">
+            <English>Jeep Wrangler (Blue)</English>
+            <Czech>Jeep Wrangler (Blau)</Czech>
+            <French>Jeep Wrangler (Azul)</French>
+            <Spanish>Jeep Wrangler (Niebieski)</Spanish>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: Polish as far as I can tell. Seems like they are all mixed up.

File: addons/realisticnames/stringtable.xml [hunk 5]
```diff
@@ -1920,5 +1920,425 @@
...
+        </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_SPAR_01_GL_blk">
+            <English>HK416A5 10" GL (Black)</English>
+            <German>HK416A5 10" GL (Černá)</German>
+            <Spanish>HK416A5 10" GL (Noir)</Spanish>
+            <Polish>HK416A5 10" GL (Negro)</Polish>
+            <Czech>HK416A5 10" GL (Чёрный)</Czech>
+            <French>HK416A5 10" GL (Czarny)</French>
+            <Russian>HK416A5 10" GL (Schwarz)</Russian>
+            <Portuguese>HK416A5 10" GL (Nero)</Portuguese>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: German. Order seems to be mixed up here too

File: addons/realisticnames/stringtable.xml [hunk 6]
```diff
@@ -1920,5 +1920,425 @@
...
+            <Italian>HK416A5 10" (Deserto)</Italian>
+        </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_SPAR_01_GL_blk">
+            <English>HK416A5 10" GL (Black)</English>
+            <German>HK416A5 10" GL (Černá)</German>
+            <Spanish>HK416A5 10" GL (Noir)</Spanish>
+            <Polish>HK416A5 10" GL (Negro)</Polish>
+            <Czech>HK416A5 10" GL (Чёрный)</Czech>
+            <French>HK416A5 10" GL (Czarny)</French>
+            <Russian>HK416A5 10" GL (Schwarz)</Russian>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: German. Order seems to be mixed up here too

**Comment by commy2** [comment]
Baer, pls fix the ordering of the Translations

**Comment by BaerMitUmlaut** [comment]
Looks like I severly screwed up on the orders - no idea how that happened.

**Comment by bux578** [comment]
Use tabler

File: addons/realisticnames/stringtable.xml [hunk 7]
```diff
@@ -1946,60 +1946,60 @@
...
         </Key>
         <Key ID="STR_ACE_RealisticNames_C_Offroad_02_unarmed_white">
             <English>Jeep Wrangler (White)</English>
-            <Czech>Jeep Wrangler (Weiss)</Czech>
-            <French>Jeep Wrangler (Blanco)</French>
-            <Spanish>Jeep Wrangler (Biały)</Spanish>
-            <Russian>Jeep Wrangler (Bílý)</Russian>
-            <Polish>Jeep Wrangler (Blanche)</Polish>
-            <German>Jeep Wrangler (Белый)</German>
+            <German>Jeep Wrangler (Weiss)</German>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: Weiß

File: addons/realisticnames/stringtable.xml [hunk 8]
```diff
@@ -2162,147 +2162,147 @@
...
-            <Czech>HK416A5 10" (Песочный)</Czech>
-            <French>HK416A5 10" (Sand)</French>
-            <Russian>HK416A5 10" (piaskowy)</Russian>
-            <Portuguese>HK416A5 10" (Sabbia)</Portuguese>
+            <Czech>HK416A5 10" (Písková)</Czech>
+            <French>HK416A5 10" (Beige)</French>
+            <Spanish>HK416A5 10" (Arena)</Spanish>
+            <Russian>HK416A5 10" (Песочный)</Russian>
+            <Polish>HK416A5 10" (Sand)</Polish>
+            <German>HK416A5 10" (piaskowy)</German>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: This is polish. I think `Sand` is the German one. `Beige` would be valid too, so I'm not sure.

File: addons/realisticnames/stringtable.xml [hunk 9]
```diff
@@ -2162,147 +2162,147 @@
...
-            <Czech>HK416A5 10" GL (Песочный)</Czech>
-            <French>HK416A5 10" GL (Sand)</French>
-            <Russian>HK416A5 10" GL (piaskowy)</Russian>
-            <Portuguese>HK416A5 10" GL (Sabbia)</Portuguese>
+            <Czech>HK416A5 10" GL (Písková)</Czech>
+            <French>HK416A5 10" GL (Beige)</French>
+            <Spanish>HK416A5 10" GL (Arena)</Spanish>
+            <Russian>HK416A5 10" GL (Песочный)</Russian>
+            <Polish>HK416A5 10" GL (Sand)</Polish>
+            <German>HK416A5 10" GL (piaskowy)</German>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: Same as above

File: addons/realisticnames/stringtable.xml [hunk 10]
```diff
@@ -2162,147 +2162,147 @@
...
-            <Czech>HK416A5 14.5" (Песочный)</Czech>
-            <French>HK416A5 14.5" (Sand)</French>
-            <Russian>HK416A5 14.5" (piaskowy)</Russian>
-            <Portuguese>HK416A5 14.5" (Sabbia)</Portuguese>
+            <Czech>HK416A5 14.5" (Písková)</Czech>
+            <French>HK416A5 14.5" (Beige)</French>
+            <Spanish>HK416A5 14.5" (Arena)</Spanish>
+            <Russian>HK416A5 14.5" (Песочный)</Russian>
+            <Polish>HK416A5 14.5" (Sand)</Polish>
+            <German>HK416A5 14.5" (piaskowy)</German>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: Same as above

File: addons/realisticnames/stringtable.xml [hunk 11]
```diff
@@ -2162,147 +2162,147 @@
...
-            <Czech>HK417A2 20" (Песочный)</Czech>
-            <French>HK417A2 20" (Sand)</French>
-            <Russian>HK417A2 20" (piaskowy)</Russian>
-            <Portuguese>HK417A2 20" (Sabbia)</Portuguese>
+            <Czech>HK417A2 20" (Písková)</Czech>
+            <French>HK417A2 20" (Beige)</French>
+            <Spanish>HK417A2 20" (Arena)</Spanish>
+            <Russian>HK417A2 20" (Песочный)</Russian>
+            <Polish>HK417A2 20" (Sand)</Polish>
+            <German>HK417A2 20" (piaskowy)</German>
```
Comment (commy2) [inline addons/realisticnames/stringtable.xml]: Same as above

**Pull request merged**